src/actions/VaccineActions.js

```diff
@@ -75,6 +84,35 @@ const scanForSensors = (dispatch, state) => {
   BleService().scanForSensors(deviceCallback);
 };
 
+const setLogFrequency = (macAddress, frequency) => async dispatch => {
+  let ok = false;
+  let error = '';
```

Comment:
I think this error is always going to be that the regex failed, unless it is overridden on L98 so maybe we can just have that as the default?

Reply:
I've managed to get 
`[BleError: Device ? is already connected]`
and 
`[BleError: Device FC:C0:EE:9D:80:A4 was disconnected]`
returned as errors - so I think worth retaining.

Reply:
But at the moment `error` is only ever assigned `e` - in the case where regex fails, it isn't assigned anything is it?

Reply:
Huh! you and your 'logic' 🙄 

thanks - 8dc3dd67